src/tutorials/0005-regular-files-api/03-exercise.md

```diff
@@ -1,3 +1,5 @@
 Add the files in your browser (available in the `files` array below) to your IPFS node using the `add` method.
 
-**Hint:** You can pass either a single `File` object or an array of `File` objects to the `add` method.
+**Hints:**
+    - You can pass either a single `File` object or an array of `File` objects to the `add` method.
+    - You need to concatenate all the results into an array because the `ipfs.add` method returns an `Async Iterable`. You can use either the `for await...of` loop or the function `all`.
```

Comment:
```suggestion
 - You need to concatenate all the results into an array because the `ipfs.add` method returns an `Async Iterable`. You can use either the `for await...of` loop or the function `all`.
```